scripts/nix/sources.json

```diff
@@ -0,0 +1,14 @@
+{
+    "nixpkgs": {
+        "branch": "release-22.11",
```

Comment:
this is ancient btw, why not 24.05?

Reply:
There is no specific reason, I used this because that's what we seem to use at mayastor-extensions.

Reply:
Changed to 24.05

Reply:
I see 23.05 currently. typo?

Reply:
No, we later decided to not go with the latest, as that might not have some packages that might be for our code.